docs/source/services/timelock_service/server_configuration.rst

```diff
@@ -0,0 +1,108 @@
+.. _timelock-server-configuration:
```

Comment:
I feel like the client and server configuration pages should go in the larger "Configuration" section of our docs. If I'm trying to configure the timelock server, the first place I'll check is probably Configuration -> Timelock -> client|server. Going to Services -> External Timelock Service -> *configuration feels less intuitive.